spec/mailers/alarm_activated_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe 'DailyReport', tipe: :mailer do
+  describe 'Send a Daily Report' do
+    let!(:user) { create(:user) }
+    let!(:alarm) { create(:alarm, user: user) }
+    let(:mail) { UserMailer.with(user: user, alarm: user.alarms.first).alarm_activated }
```

Comment:
Isn't it the same if you use the alarm that use created already?
`let(:mail) { UserMailer.with(user: user, alarm: alarm).alarm_activated }`

Reply:
Ok, I think this is a step in the process. Take a look at this: https://relishapp.com/rspec/rspec-core/v/3-10/docs/subject